dist/DescriptionList/DescriptionDetails.d.ts

```diff
@@ -0,0 +1,2 @@
+export default DescriptionDetails;
+declare const DescriptionDetails: any;
```

Comment:
`DescriptionList` and friends probably shouldn't be `any`, so they're good candidates to convert to TS soon.